.github/workflows/test-e2e.yml

```diff
@@ -155,7 +155,7 @@ jobs:
           done
       - name: Checkout template-operator
         uses: actions/checkout@v3
-        if: ${{ matrix.e2e-test != 'watcher-enqueue' && matrix.e2e-test != 'kyma-metrics'}}
+        if: ${{ matrix.e2e-test != 'watcher-enqueue' && matrix.e2e-test != 'kyma-metrics' && matrix.e2e-test != 'module-upgrade'}}
```

Comment:
let's remove this if condition, not make too much sense maintain this template-operator checkout, we can allow all tests have this checkout step, even not in use later.